heron/healthmgr/src/java/org/apache/heron/healthmgr/HealthManager.java

```diff
@@ -153,7 +152,6 @@ public HealthManager(Config config, AbstractModule baseModule) {
   }
 
   public static void main(String[] args) throws Exception {
-    Slf4jUtils.installSLF4JBridge();
```

Comment:
This is no longer needed?

Reply:
Yes. In HealthManger , there was a problem and it was removed.